website/docs/guides/orchestration/custom-cicd-pipelines/3-dbt-cloud-job-on-merge.md

```diff
@@ -283,6 +305,42 @@ run-dbt-cloud-job:
 </Tabs>
 
 
+</TabItem>
+<TabItem value="ado">
+
+For this new job, open the existing Azure pipeline you created above and select the *Edit* button. We'll want to edit the corresponding Azure pipeline YAML file with the appropriate configuration, instead of the starter code, along with including a `variables` section to pass in the required variables.
+
+Copy the below YAML file into your Azure pipeline and update the variables below to match your setup based on the comments in the file. It's worth noting that we changed the `trigger` section so that it will run **only** when there are pushes to a branch named `master` (like a PR merged to your master branch). Have a read through [Azure's docs](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops) on these filters for additional use cases.
```

Comment:
```suggestion
Copy the below YAML file into your Azure pipeline and update the variables below to match your setup based on the comments in the file. It's worth noting that we changed the `trigger` section so that it will run **only** when there are pushes to a branch named `master` (like a PR merged to your master branch). 

Read through [Azure's docs](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops) on these filters for additional use cases.
```